src/manage_sql_secinfo.c

```diff
@@ -3438,6 +3415,39 @@ update_scap_end ()
       update_cvss_dfn_cert (1, last_cert_update, last_scap_update);
       update_cvss_cert_bund (1, last_cert_update, last_scap_update);
     }
+}
+
+/**
+ * @brief Finish scap update.
+ *
+ * @return 0 success, -1 error.
+ */
+static int
+update_scap_end ()
```

Comment:
I think this function is no longer returning -1. It returns 0 in all cases now. Do you think it make sense to change it to return void ? If so, the callers would need to be updated accordingly to return 0 on success.

```suggestion
 */
static void
update_scap_end ()
```